src/cmd-buildextend-metal

```diff
@@ -185,15 +189,18 @@ ostree --repo="${tmprepo}" cat "${commit}" "${target_moduledir}/config" > tmp/ta
 # We need to support kernels that don't have this enabled yet, including RHEL8
 # and current Fedora 31.
 # https://bugzilla.redhat.com/show_bug.cgi?id=1765933
-# We unconditonally enable fs-verity for /boot if we have it,
-# because...integrity there is useful a bit and it will help
-# us prove this out even if we're not using it for the rootfs.
-if grep -Eq '^CONFIG_FS_VERITY=y' tmp/target-kernel.config; then
-    disk_args+=("--boot-verity")
-else
-    echo 'NOTE: Missing CONFIG_FS_VERITY from target kernel config'
-    sleep 1
-fi
+case "${bootfs_type}" in
```

Comment:
Maybe a comment here to mention the `coreos-installer` + old machine path so we remember this when we revisit?